{
  "exploitdb": [],
  "itw": [
    {
      "id": "CVE-2018-9918",
      "description": "libqpdf.a in QPDF through 8.0.2 mishandles certain \"expected dictionary key but found non-name object\" cases, allowing remote attackers to cause a denial of service (stack exhaustion), related to the QPDFObjectHandle and QPDF_Dictionary classes, because nesting in direct objects is not restricted.",
      "referenceURL": "https://nvd.nist.gov/vuln/detail/CVE-2018-9918",
      "reportURL": "https://github.com/qpdf/qpdf/issues/202",
      "timeStamp": "2019-10-03T00:03:00Z"
    }
  ],
  "trickest": [
    {
      "id": "CVE-2018-9918",
      "url": "https://github.com/qpdf/qpdf/issues/202",
      "added_at": "2022-02-11T19:41:50Z"
    }
  ],
  "nomisec": [],
  "nuclei": [],
  "metasploit": [],
  "holloways": [],
  "openpoc": [
    {
      "cve": "CVE-2018-9918",
      "url": "https://github.com/qpdf/qpdf/issues/202",
      "added_at": "2019-10-03T00:03:00Z",
      "score": 0.4,
      "template_for": ""
    }
  ]
}
